examples/README.md

```diff
@@ -42,9 +42,11 @@ The [Blockly Codelabs](https://blocklycodelabs.dev/) refer to this example code.
 - [``blockly-vue3-sample``](blockly-vue3/): Blockly in a Vue3 project, defines a Vue Blockly Component.
 - [``blockly-parcel``](blockly-parcel/): Using Blockly with Parcel.
 
+
 ### Real-time Collaboration
 
 - [``blockly-rtc``](blockly-rtc/): Real-time collaboration environment on top of the Blockly framework.
+- [``sample-app``](sample-app/):This app illustrates how to use Blockly together with common programming tools.
```

Comment:
Hello! Thanks for noticing that this was missing :D

Some thoughts:
1) We would want this to go under "Integrating Blockly" instead of "Real-time Collaboration".
2) Can you list some of the common programming tools? E.g. webpack.

Important: Can you also sign the CLA? I'm not able to merge this until you give us permission to use your changes =)

Thank you again for putting up this fix!